### ISSUE TITLE
Patchmatrix stopped working after jack2 last update

### ISSUE THREAD

**Issue by Yrds** [OpenMusicKontrollers/patchmatrix#13]
Everytime i launch patchmatrix i am getting the following error:

```
yuri@archlinux ~> patchmatrix
patchmatrix: Symbol `JACK_METADATA_PRETTY_NAME' has different size in shared object, consider re-linking
patchmatrix 0.20.0
Copyright (c) 2016-2020 Hanspeter Portner (dev@open-music-kontrollers.ch)
Released under Artistic License 2.0 by Open Music Kontrollers
Using visual 0x21: R=8 G=8 B=8 A=8 D=24 DOUBLE=1 SAMPLES=0
fish: 'patchmatrix' terminated by signal SIGSEGV (Address boundary error)
```

Jack2 version: 1.9.15-1

~~I thought it was jack fault, but other programs like calf and qjackctl are still working normally.~~

Please let me know if you need more information

Edit:

In fact, i tested qjackctl before jack update. It also is giving sigsegv. I will open a issue on jack2 issue tracker because it's affecting more than one application.

**Comment by ventosus** [comment]
hmm, can confirm, Arch package crashes with new jack2, works fine with jack1, though.

patchmatrix compiled against jack2 (1.9.15-1) runs fine on jack2
patchmatrix compiled against jack1 (0.125.0-9) crashes on jack2

The headers apparently have changed/are different, maybe that's the cause ? Maybe @falktk can shed some light ?

jack2:
    [hp@melamori ~]$ grep METADATA_PRETTY_NAME /usr/include/jack/*.h
    /usr/include/jack/metadata.h:extern const char JACK_METADATA_PRETTY_NAME[];

jack1:
    [hp@melamori ~]$ grep METADATA_PRETTY_NAME /usr/include/jack/*.h
    /usr/include/jack/metadata.h:extern const char* JACK_METADATA_PRETTY_NAME;

**Comment by Yrds** [comment]
The problem was fixed in jack2 1.9.16(next release), where the headers was reverted to the old style.

Thanks for the support. @ventosus
